docs/validators/system-requirements.md

```diff
@@ -17,26 +17,21 @@ Nodes are computers running the MultiversX software, so they contribute to the M
 
 ## **Minimum System Requirements for running 1 MultiversX Node**
 
-- 4 x dedicated/physical CPUs, either Intel or AMD, with the `SSE4.1` and `SSE4.2` flags (use [lscpu](https://manpages.ubuntu.com/manpages/trusty/man1/lscpu.1.html) to verify)
+- 4 x dedicated/physical CPUs, either Intel or AMD, **with `SSE4.1` and `SSE4.2` flags** (use [lscpu](https://manpages.ubuntu.com/manpages/trusty/man1/lscpu.1.html) to verify)
 - 8 GB RAM
 - 200 GB SSD
 - 100 Mbit/s always-on internet connection, at least 4 TB/month data plan
 - Linux OS (Ubuntu 22.04 recommended) / MacOS
 
-:::tip
-Support for ARM processors (e.g. for Raspberry Pi) will come in the future, pending third-party issues.
-:::
-
 :::caution
-If the system chosen to host the node is a VPS, the host should have dedicated CPUs. This is mandatory. Using shared CPUs can hinder your node's performance that will result in a decrease of node's rating and eventually the node might get jailed.
+1. The CPUs must be `SSE4.1` and `SSE4.2` capable, otherwise the node won't be able to use the Wasmer 2 VM available through the VM 1.5 (and above) and the node will not be able to sync blocks from the network.
+2. If the system chosen to host the node is a VPS, the host should have dedicated CPUs. This is mandatory. Using shared CPUs can hinder your node's performance that will result in a decrease of node's rating and eventually the node might get jailed.
+3. In case a decision to run multiple MultiversX Nodes on the same machine is chosen, the host running those nodes should have at least the minimum system requirements multiplied by the number of nodes running on that host.
```

Comment:
~~In case a decision to~~ **If you** run multiple MultiversX Nodes on the same machine ~~is chosen~~ 

should have **the specs at least equal** to ~~at least~~ the minimum system requirements